frontend/src/main/pages/Admin/AdminJobsPage.js

```diff
@@ -156,6 +156,12 @@ const AdminJobsPage = () => {
       <h2 className="p-3">Job Status</h2>
 
       <JobsTable jobs={jobs} />
+
+      <div className="mt-3">
+        <button className="btn btn-danger" data-testid="purgeJobLogButton">
+          {"Purge Job Log"}
+        </button>
+      </div>
```

Comment:
I'm confused.. does this button actually do anything?   A button typically has some kind of action associated with it.

This seems to be putting a button on the page that doesn't do anything at all.

Instead, it should be tied to a backend post call or delete call that actually purges the job log. 

Adding a button that doesn't do anything isn't particularly helpful.